src/sentry/static/sentry/app/views/eventsV2/chartFooter.tsx

```diff
@@ -29,11 +30,19 @@ type Props = {
   hoverState: TooltipData;
 };
 
-function formatValue(val: string | number, columnName: string) {
+function formatValue(val: string | number, columnName: string, itemName: string) {
   // Extract metadata from the columnName so we can format the
   // value appropriately.
   const columnData = decodeColumnOrder([{field: columnName}])[0];
 
+  if (itemName === t('Release')) {
+    return (
+      <Value>
+        <Version version={val as string} anchor={false} withPackage />
```

Comment:
```suggestion
        <Version version={String(val)} anchor={false} withPackage />
```

Reply:
as you are not checking if the val is typeof string, maybe it would be better to use `String(val)` instead of casting . Please what do you think?